app/templates/_view_suppliers_edit_list.html

```diff
@@ -1,10 +1,12 @@
 {% set field_headings = [
   "Supplier Name"
-] if current_user.has_role('admin-ccs-sourcing') else [
+] + ([
   "Supplier Name",
```

Comment:
I think this causes `Supplier Name` to appear twice in the table heading...

Reply:
Doh. This was a late addition.